lib/ledger_sync/adaptors/netsuite/customer/serializer.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module LedgerSync
+  module Adaptors
+    module NetSuite
+      module Customer
+        class Serializer < NetSuite::Serializer
+          attribute :companyName,
+                    resource_attribute: :name
+
+          attribute :entityId,
+                    resource_attribute: :external_id
+
+          attribute :externalId,
+                    resource_attribute: :external_id
+
+          attribute :email
+
+          attribute :phone,
+                    resource_attribute: :phone_number
+
+          # attribute :firstName,
+          #           resource_attribute: :first_name
+
+          # attribute :lastName,
+          #           resource_attribute: :last_name
+
```

Comment:
Are we leaving these commented out blocks in?

Reply:
These were in the originals, so I left them.  Not sure why though.  I figured I'd leave them for now until we decide to add/remove.